sphinx/source/metrics.rst

```diff
@@ -0,0 +1,15 @@
+Metrics
+======
```

Comment:
@vivekmig, updated RSTs as well. I guess this should be sufficient ?

Reply:
I think we need to also add metrics to index.rst so that this is included in the main API list. You can also run ./scripts/build_docs locally to preview the website and double-check the docs look ok if you want.